pkg/printers/internalversion/printers.go

```diff
@@ -2624,6 +2637,57 @@ func printPriorityLevelConfigurationList(list *flowcontrol.PriorityLevelConfigur
 	return rows, nil
 }
 
+func printClusterCIDRConfig(obj *networking.ClusterCIDRConfig, options printers.GenerateOptions) ([]metav1.TableRow, error) {
+	row := metav1.TableRow{
+		Object: runtime.RawExtension{Object: obj},
+	}
+	ipv4 := "<none>"
+	ipv6 := "<none>"
+
+	if obj.Spec.IPv4CIDR != "" {
+		ipv4 = obj.Spec.IPv4CIDR
+	}
+	if obj.Spec.IPv6CIDR != "" {
+		ipv6 = obj.Spec.IPv6CIDR
+	}
+
+	row.Cells = append(row.Cells, obj.Name, fmt.Sprint(obj.Spec.PerNodeHostBits), ipv4, ipv6, translateTimestampSince(obj.CreationTimestamp))
+	if options.Wide {
+		nodeSelector := "<none>"
+		if obj.Spec.NodeSelector != nil {
```

Comment:
can you use `metav1.FormatLabelSelector` (or some such equivalent) rather than defining our own? Feels like this should be a utility.

Reply:
I couldn't find a method for NodeSelector, should this be moved to metav1?